scripts/pin_the_slow_way.py

```diff
@@ -41,7 +41,7 @@
           'libblitz': 'libblitz 0.10|0.10.*',
           'libmatio': 'libmatio 1.5.*',
           'libnetcdf': 'libnetcdf 4.4.*',
-          'libpng': 'libpng >=1.6.21,<1.7',
+          'libpng': 'libpng >=1.6.23,<1.7',
```

Comment:
According to this [chart]( http://abi-laboratory.pro/tracker/timeline/libpng/ ), this should be `22`.